ipatests/test_integration/test_sssd.py

```diff
@@ -108,3 +113,45 @@ def test_auth_cache_enabled_when_configured(self, user):
             assert self.is_auth_cached(self.users[user])
             time.sleep(10)
             assert not self.is_auth_cached(self.users[user])
+
+    @contextmanager
+    def filter_user_setup(self, user):
```

Comment:
Do you think setup should be in separate function. It looks rather simple and can be incorporated in test function.

Reply:
yes i think it should be in separate function, there will be need of this function where we want to test user without domain.